apps/_compability_app/schema/User.js

```diff
@@ -1,9 +1,20 @@
 const { Wysiwyg } = require('@keystonejs/fields-wysiwyg-tinymce')
-const { User: BaseUser, ForgotPasswordAction, ForgotPasswordService, RegisterService } = require('@core/keystone/schemas/User')
+const {
+    User: BaseUser,
+    ForgotPasswordAction,
+    ForgotPasswordService,
+    RegisterService,
+} = require('@core/keystone/schemas/User')
+const { Relationship } = require('@keystonejs/fields')
 
 const User = BaseUser._override({
     fields: {
         aboutMyself: { type: Wysiwyg },
+        tests: {
```

Comment:
No need to connect tests to users. Just results

Reply:
Fixed; Results relation will be added later.